xchange-stream-bitfinex/src/test/java/info/bitrich/xchangestream/bitfinex/BitfinexManualExample.java

```diff
@@ -28,7 +28,7 @@ private static void rateLimit() {
       LOG.warn("Bitfinex connection throttle control has been interrupted");
     }
   }
-
+  @SuppressWarnings("deprecation")
```

Comment:
Can we not remove the use of the untrusted API?

Not sure suppressing the warning is the right thing to do here - a warning is warranted.